apipe/store/datasets/bnetza_mastr_biomass_region/scripts/create.py

```diff
@@ -3,15 +3,8 @@
 
 from apipe.config import GLOBAL_CONFIG
 from apipe.scripts.datasets import mastr
-from apipe.scripts.geo import (
-    overlay,
-    rename_filter_attributes,
-    write_geofile,
-)
-from apipe.store.utils import (
-    PATH_TO_REGION_DISTRICTS_GPKG,
-    get_names_from_nuts,
-)
+from apipe.scripts.geo import overlay, rename_filter_attributes, write_geofile
+from apipe.store.utils import PATH_TO_REGION_DISTRICTS_GPKG, get_names_from_nuts
```

Comment:
Here I was wondering why the formatting had changed. Has the black changed?

Reply:
I haven't changed the config, so it might just have not been blacked before..
As the line limit of 80 chars is met, it's fine.